rdr_service/resource/schemas/participant.py

```diff
@@ -122,6 +130,19 @@ class Meta:
         pii_filter = {}  # dict(field: lambda function).
 
 
+class SexualOrientationSchema(Schema):
```

Comment:
Do you want to add a test case to the `tests/resource_tests/schema_tests/test_resource_schemas.py` file, since we're creating both BQ and the resource schemas for this?

Reply:
Done